src/Handler/CurlFactory.php

```diff
@@ -484,7 +487,7 @@ private function applyHandlerOptions(EasyHandle $easy, array &$conf)
                 if (is_resource($args[0])) {
                     array_shift($args);
                 }
-                call_user_func_array($progress, $args);
+                return call_user_func_array($progress, $args);
```

Comment:
Why is this needed?

Reply:
The `CURLOPT_PROGRESSFUNCTION` option return a non-zero value to abort the transfer.

https://www.php.net/manual/en/function.curl-setopt.php

> Return a non-zero value to abort the transfer. In which case, the transfer will set a CURLE_ABORTED_BY_CALLBACK error.